pkg/sql/logictest/testdata/logic_test/grant_on_all_tables_in_schema

```diff
@@ -94,3 +94,21 @@ query TTTTT colnames
 SHOW GRANTS FOR testuser, testuser2
 ----
 database_name  schema_name  relation_name  grantee  privilege_type
+
+# Verify that the database name is resolved correctly if specified.
+statement ok
+CREATE DATABASE otherdb
+
+statement ok
+CREATE TABLE otherdb.public.tbl (a int)
+
+statement ok
+GRANT SELECT ON ALL TABLES IN SCHEMA otherdb.public TO testuser
+
+query TTTTTB colnames
+SHOW GRANTS ON TABLE otherdb.public.tbl
```

Comment:
want to do a test for multiple different tables at once, just in case?
(looks like we take multiple tables here)